frontend/src/pages/projects/components/GenericHorizontalBar.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {
+  Tabs,
+  Tab,
+  TabTitleIcon,
+  TabTitleText,
+  TabContent,
+  TabContentBody,
+  PageSection,
+  PageSectionVariants,
+} from '@patternfly/react-core';
+import { SVGIconProps } from '@patternfly/react-icons/dist/esm/createIcon';
+
+type GenericHorizontalBarProps = {
+  sections: {
+    title: string;
+    component: React.ReactNode;
+    icon?: React.ReactElement<React.ComponentClass<SVGIconProps>>;
+    id?: string;
+  }[];
+};
+
+const GenericHorizontalBar: React.FC<GenericHorizontalBarProps> = ({ sections }) => {
+  const [activeTabKey, setActiveTabKey] = React.useState<string | number | undefined>(
+    sections[0].title || undefined,
+  );
+
+  return (
+    <>
+      <PageSection
```

Comment:
I'm not 100% confident with this component, I followed the example of Tabs in patternfly and this is the most common patter they are using, which fixes issues with page fillings. But this is what is breaking the Jumping Links in DS Projects.